pmm-app/src/pmm-check/components/AllChecksTab/AllChecksTab.tsx

```diff
@@ -8,15 +8,28 @@ import { getStyles as getTableStyles } from 'pmm-check/components/Table/Table.st
 import * as checkPanelStyles from 'pmm-check/CheckPanel.styles';
 import { Messages } from './AllChecksTab.messages';
 import * as styles from './AllChecksTab.styles';
-
+import { FetchChecks } from './types';
+import { CheckTableRow } from './CheckTableRow';
 
 export const AllChecksTab: FC = () => {
   const [fetchChecksPending, setFetchChecksPending] = useState(false);
   const [checks, setChecks] = useState<CheckDetails[] | undefined>();
   const theme = useTheme();
   const tableStyles = getTableStyles(theme);
 
-  const fetchChecks = async (): Promise<void> => {
+  const updateUI = (check: CheckDetails) => {
+    const { name, enabled } = check;
+
+    setChecks((oldChecks) => oldChecks?.map((c) => {
```

Comment:
maybe it make sense to rename `c` to `oldCheck`

Reply:
Good idea, done :)